app/elements/coder/profile/profile-page.html

```diff
@@ -119,7 +126,13 @@ <h3 class="card-header">Teams</h3>
 				</template>
 			</paper-material>
 			<paper-material class="right" elevation="1">
-				<h3 class="card-header">Challenges</h3>
+				<template is="dom-if" if="[[isCurrentUser]]">
```

Comment:
the platform stuff is not implemented for this and we would need another table for displaying that data so, the dom-if is ok.